src/foam/u2/grid/seutil.js

```diff
@@ -0,0 +1,396 @@
+console.log("Loading SE.util");
+var SE = {};
+
+
+SE.util = {};
+/*
+ *(fucntion(ns) { ns.fct = function(){...} }_)(SE.util)
+ * is a convenient way to decorate and assign functions to SE.util
+ *
+*/
+(function(ns) {
+    /* local ServiceEcho functions */
+    ns.getRandomInt = function(max, min){
```

Comment:
It seems odd that you would specify the min after the max. You could check arguments.length to determine if the user has specified: (max) or (min, max).